### PR TITLE
Add support for Seeed T1000-E boards

### PR DESCRIPTION
**Pull request by markbirss**
This PR adds support for Seeed nRF82540 T1000E boards

Confirmed Vendor ID: 0x2886
Confirmed Product ID: 0x0057

https://the-sz.com/products/usbid/index.php?v=0x2886&p=0x0057&n=Seeed

UF2_BOARD_ID: nRF52840-T1000-E

-----------

## Description of Change

Adds new support

### CODE REVIEW

**Comment by markbirss** [comment]
Kindly review and comment on my PR

**Review by hathach — CHANGES_REQUESTED** [review]

**Comment by markbirss** [comment]
@hathach I have updated with changes required

File: src/boards/t1000_e/board.h [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#define USB_DESC_VID            0x2886
+#define USB_DESC_UF2_PID        0x0057
+#define USB_DESC_CDC_ONLY_PID   0x0057
+
+//--------------------------------------------------------------------+
+// UF2
+//--------------------------------------------------------------------+
+#define UF2_PRODUCT_NAME        "Seeed T1000-E for Meshtastic"
+#define UF2_VOLUME_LABEL        "T1000-E"
+#define UF2_BOARD_ID            "Seeed-nRF52840-T1000-E"
```
[the remark this answers is not in the document]
Comment (markbirss) [reply src/boards/t1000_e/board.h]: updated

**Review by hathach — APPROVED** [review]
look good, thank you

**Pull request merged**